client/src/pages/Signup.js

```diff
@@ -0,0 +1,113 @@
+import { Box, Grid, Typography, TextField, Button } from '@material-ui/core'
+import React, { useState } from 'react'
+import { Link } from 'react-router-dom'
+import useStyles from '../styles'
+import green from '@material-ui/core/colors/green'
+import { useForm } from 'react-hook-form'
+
+const Register = () => {
+    const classes = useStyles()
+    const { register, handleSubmit, formState: { errors }, watch } = useForm()
+
+    const password = watch('password')
+
+    const onSubmit = (formData) => {
+        console.log(formData)
+    }
+
+    return (
+        <Grid container item direction='column' xs={12} spacing={6}>
+            <Grid item className={classes.inlineCenter} xs={12}>
+                <Typography className={classes.formHeaderTitle}>Create an account</Typography>
+                <hr size="0.5" width="3%" color={green['A400']}></hr>
```

Comment:
I believe that Instances of colors like this should probably be set/referenced from the theme file.
According to the notes, the "primary" color code is "#69E781". I updated this in the theme when I was putting together the skeleton.

I have not tried to import them outside of makeStyles, but You should be able to import theme and access it with:
```
color = ${theme.palette.primary.main}
```